xblockmufi/private/view.html

```diff
@@ -1,6 +1,6 @@
 <div class="xblockmufi_block">
-    <h2>{display_name}</h2>
-    <div class='your_answer' style='display:none'>{your_answer_label}</div>
+    <h2>{{ display_name }}</h2>
+    <div class='your_answer' style='display:none'>{{ your_answer_label }}</div>
```

Comment:
since we're adding the `nodisplay` class, let's use that to hide the answer initially, then edit the JS that display's the answer to remove the class.

Reply:
this way we can remove the inline style

Reply:
So 2 questions here:

1. For using the `nodisplay` class, do you want me to say then `<div class='your_answer nodisplay'>`?

2. Are you telling me to edit these lines in the JS: ```     function show_answer() {
         your_answer.css('display','block');
         expert_answer.css('display','block');
         submit_button.val('Resubmit');
 
     }
 
     function reset_answer() {
         your_answer.css('display','none');
         expert_answer.css('display','none');
         submit_button.val('Submit and Compare');
     }```
So should I delete the `your_answer` and `expert_answer` lines from both, or should I attach a conditional?